accounts-journey/src/androidTest/java/app_common/BasePage.kt

```diff
@@ -0,0 +1,15 @@
+package app_common
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.ViewInteraction
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import com.backbase.accounts_journey.R
+
+open class BasePage {
```

Comment:
Should this be BasePage or BaseScreen?

Reply:
You are right, I updated everywhere to `Screen`